### DISCUSSION TITLE
Issue with interReactiveTransportFoam

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#15] [Q&A] [answered]
Hello all,

Hope everything is fine with you.

I am encountering an issue with interReactiveTransportFoam. The tutorial case works well however when I try to apply my own case, the first few loops works good but then there is this following issue appearing and aborting the simulation:
''PIMPLE: iteration 1
slurmstepd: error: *** STEP 11679913.0 ON cdr463 CANCELLED AT 2023-09-08T02:24:4                         3 ***
srun: error: cdr726: task 17: Floating point exception (core dumped)
''

I tried to reduce the courant number, switch decomposition method (because I run it on parallel), but it didn't work.
Did anyone encounter this issue or have an idea how to solve it ?

Any help would be greatly appreciated.

Sincerely,
Santhosh

**Comment by julienmaes** [comment, marked as answer]
Hi Santhosh, what did you modify from the original tutorial to make your own case?
